backend/src/main/java/kernel360/techpick/core/model/folder/Folder.java

```diff
@@ -62,7 +61,7 @@ public void updateParentFolder(Folder parentFolder) {
 	}
 
 	// TODO: 엔티티 사용자가 정적 팩토리 메소드로 필요한 함수를 구현 하세요
-	public static Folder create(String name, Folder parentFolder, FolderType folderType, User user) {
-		return new Folder(name, parentFolder, folderType, user);
+	public static Folder create(String name, FolderType folderType, User user) {
+		return new Folder(name, folderType, user);
```

Comment:
p2: 생성자에서 parentFolder를 빼지 말고, 아래처럼 하는게 어떨지?
```suggestion
	createFolderWithNoParent(String name, FolderType folderType, User user) {
        return new Folder(name, null, folderType, user);
    }
```